javascript/i18n/phonenumbers/phonenumberutil.js

```diff
@@ -2430,10 +2430,13 @@ i18n.phonenumbers.PhoneNumberUtil.prototype.
 i18n.phonenumbers.PhoneNumberUtil.prototype.getNationalSignificantNumber =
     function(number) {
 
-  // If leading zero(s) have been set, we prefix this now. Note this is not a
-  // national prefix.
+  if (!number.hasNationalNumber()) {
+    return '';
+  }
   /** @type {string} */
   var nationalNumber = '' + number.getNationalNumber();
+  // If leading zero(s) have been set, we prefix this now. Note these are not a
```

Comment:
these are --> this is?
alternatively, change the whole thing:

`we prefix these now. Note these do not make up a national prefix.`

this -> these
there are not a -> these do not make up a